practices/graded-browser-support.md

```diff
@@ -1,67 +1,90 @@
+This technique is documented in [Cutting the Mustard with Media queries](https://www.sitepoint.com/cutting-the-mustard-with-css-media-queries/). The specific media queries we use are based upon [CSS Only Mustard Cut](https://github.com/Fall-Back/CSS-Mustard-Cut), with a preference towards combining them into one rather than separating them out into multiple `<link>` elements. Note that you **cannot** add line breaks to the media query if they are combined.
 
-In order to check this, we use [window.matchMedia](https://developer.mozilla.org/en/docs/Web/API/Window/matchMedia).
+We couple the loading of JavaScript to the loading of the enhanced CSS. We use the [window.matchMedia](https://developer.mozilla.org/en/docs/Web/API/Window/matchMedia) API to detect when the browser loads the CSS within the media query. Once the enhanced CSS loads, this will cause the JavaScript code to load too.
 
 ```javascript
 (function() {
-    var linkEl = document.querySelector('link');
-    if (window.matchMedia && window.matchMedia(linkEl.media).matches) {
-        var script = document.createElement('script');
-        script.src = 'your-script.js';
-        script.async = true;
-        document.body.appendChild(script);
-    }
+  var linkEl = document.querySelector('link');
+  if (window.matchMedia && window.matchMedia(linkEl.media).matches) {
+    var script = document.createElement('script');
+    script.src = 'your-script.js';
+    script.async = true;
+    document.body.appendChild(script);
+  }
 })();
 ```
 
-The javascript code outlined above will fail in Internet Explorer if there is more than one link element. You can get around this by targeting the specific link element using a class or ID.
+Please note that this JavaScript code will fail in Internet Explorer if there is more than one link element. You can get around this by targeting the specific link element using a class or ID.
+
+### Caveats
+
+#### Internet Explorer 10 support
+
+As on January 2017, we consider Internet Explorer 10 as grade-C. This is mainly due to its lack of support for certain modern features plus a very low usage. IE10 also stopped receiving security updates in January 2016 from its manufacturer.
+
+Unfortunately, both IE10 and IE11 support the same CSS media queries, which means that it's not possible to use a CSS-only method of differentiating between both browsers.
+
+This means that even if we consider IE10 a Grade-C browser, we currently serve IE10 users the full experience.
+
+#### Browsers without TLS 1.2 support
+
+As of June 2018, all our sites are served through HTTPS using the [TLS 1.2 cryptographic protocol](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.2) or newer. This means that users of browsers that don't support TLS 1.2 (e.g. Safari on iOS 4) or that have support for TLS 1.2 disabled (e.g. Internet Explorer on Windows 7) may not be able to access our sites. We consider that this is required in order to keep our users secure.
+
```

Comment:
Maybe add this?

> Note that for IE 8, 9 & 10 on Windows 7 & 8, users can enable TLS 1.2 support themselves: _Tools > Internet Options > Advanced > Settings (Security Section)_. 

It might help a user out there... somewhere...

Reply:
I don't feel like that's in the scope of this document, which is mostly browser support and its implementation details? I wouldn't expect users to find this document.

Additionally, there are other browser & O.S. combos that support TLS 1.2 but don't have the support enabled by default, so this is not specific about IE10. Shirley we don't want to add all the possible ways to enable TLS 1.2 in all the possible browsers?

Reply:
> I don't feel like that's in the scope of this document, which is mostly browser support and its implementation details? I wouldn't expect users to find this document.

True, but if say customer care is contacted by a user there is some advice for them. The playbook is useful for a reference we can point people at, as well as a thing that can be discovered.

> Additionally, there are other browser & O.S. combos that support TLS 1.2 but don't have the support enabled by default, so this is not specific about IE10. Shirley we don't want to add all the possible ways to enable TLS 1.2 in all the possible browsers?

No, naturally, but IE users are going to be the biggest chunk of those users, and I don't think it does any harm to offer help for these poor souls.

Reply:
> True, but if say customer care is contacted by a user there is some advice for them. The playbook is useful for a reference we can point people at, as well as a thing that can be discovered.

Sure, though I think that advice for customer care would better belong to our intranet than here.

Reply:
I'm up for adding the IE10 TLS config here. It _could_ help the bulk of users with an issue, and acts as a public place to point customer care and customers.  

I guess it shows that we're aware that an exception to our "100% coverage of all users" philosophy is caused by an erroneous browser setting and not by us doing something "wrong".